### ISSUE TITLE
Site Importer: options other than WordPress display a blank page.

### ISSUE THREAD

**Issue by worldomonation** [Automattic/wp-calypso#55251]
### Steps to Reproduce

Prerequisite. log in as a testing account user. I was shown a different importer when logging in with my personal account.

1. start at /home.
2. from the sidebar, navigate to Tools > Import.
3. confirm the following options are present: [WordPress, Blogger, Medium, SquareSpace, Wix]
4. click on an option other than WordPress.

### What I expected to happen

- as with clicking the WordPress option, the importer should present the next step in a screen.

### What actually happened

- an entirely blank screen is shown to user.

### Context

While developing automated tests for Playwright project.

### Operating System

macOS

### OS Version

11.4

### Browser

Chrome/Chromium

### Browser Version

91.0.864.37 (Official build) (arm64)

### Simple, Atomic or both?

Simple

### Console and/or error logs

```
27502.0a665b01c528ab9620e3.min.js:2 TypeError: Cannot read property 'validate' of null
    at it.validateUrl (import.39fc2217e4ca8f483726.min.js:1)
    at it.render (import.39fc2217e4ca8f483726.min.js:1)
    at Wi (27502.0a665b01c528ab9620e3.min.js:2)
    at Ui (27502.0a665b01c528ab9620e3.min.js:2)
    at wa (27502.0a665b01c528ab9620e3.min.js:2)
    at vl (27502.0a665b01c528ab9620e3.min.js:2)
    at ml (27502.0a665b01c528ab9620e3.min.js:2)
    at al (27502.0a665b01c528ab9620e3.min.js:2)
    at 27502.0a665b01c528ab9620e3.min.js:2
    at t.unstable_runWithPriority (27502.0a665b01c528ab9620e3.min.js:2)
oa @ 27502.0a665b01c528ab9620e3.min.js:2
n.callback @ 27502.0a665b01c528ab9620e3.min.js:2
ys @ 27502.0a665b01c528ab9620e3.min.js:2
ua @ 27502.0a665b01c528ab9620e3.min.js:2
jl @ 27502.0a665b01c528ab9620e3.min.js:2
t.unstable_runWithPriority @ 27502.0a665b01c528ab9620e3.min.js:2
qo @ 27502.0a665b01c528ab9620e3.min.js:2
_l @ 27502.0a665b01c528ab9620e3.min.js:2
al @ 27502.0a665b01c528ab9620e3.min.js:2
(anonymous) @ 27502.0a665b01c528ab9620e3.min.js:2
t.unstable_runWithPriority @ 27502.0a665b01c528ab9620e3.min.js:2
qo @ 27502.0a665b01c528ab9620e3.min.js:2
Yo @ 27502.0a665b01c528ab9620e3.min.js:2
Go @ 27502.0a665b01c528ab9620e3.min.js:2
I @ 27502.0a665b01c528ab9620e3.min.js:2
Gt @ 27502.0a665b01c528ab9620e3.min.js:2
61065.b28b48263e1affaf1a60.min.js:2 [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
(anonymous) @ 61065.b28b48263e1affaf1a60.min.js:2
n @ 61065.b28b48263e1affaf1a60.min.js:2
o @ 61065.b28b48263e1affaf1a60.min.js:2
v @ 61065.b28b48263e1affaf1a60.min.js:2
b @ 61065.b28b48263e1affaf1a60.min.js:2
a @ 61065.b28b48263e1affaf1a60.min.js:2
error (async)
(anonymous) @ 61065.b28b48263e1affaf1a60.min.js:2
c.subscribe @ 61065.b28b48263e1affaf1a60.min.js:2
x @ entry-main.726e9d7664906f646de0.min.js:2
(anonymous) @ entry-main.726e9d7664906f646de0.min.js:2
Lt @ entry-main.726e9d7664906f646de0.min.js:2
(anonymous) @ entry-main.726e9d7664906f646de0.min.js:2
Promise.then (async)
Ct @ entry-main.726e9d7664906f646de0.min.js:2
(anonymous) @ entry-main.726e9d7664906f646de0.min.js:2
async function (async)
(anonymous) @ entry-main.726e9d7664906f646de0.min.js:2
window.AppBoot @ entry-main.726e9d7664906f646de0.min.js:2
(anonymous) @ e2eflowtesting3.wordpress.com?retry=1:38
27502.0a665b01c528ab9620e3.min.js:2 Uncaught TypeError: Cannot read property 'validate' of null
    at it.validateUrl (import.39fc2217e4ca8f483726.min.js:1)
    at it.render (import.39fc2217e4ca8f483726.min.js:1)
    at Wi (27502.0a665b01c528ab9620e3.min.js:2)
    at Ui (27502.0a665b01c528ab9620e3.min.js:2)
    at wa (27502.0a665b01c528ab9620e3.min.js:2)
    at vl (27502.0a665b01c528ab9620e3.min.js:2)
    at ml (27502.0a665b01c528ab9620e3.min.js:2)
    at al (27502.0a665b01c528ab9620e3.min.js:2)
    at 27502.0a665b01c528ab9620e3.min.js:2
    at t.unstable_runWithPriority (27502.0a665b01c528ab9620e3.min.js:2)
```

### Level of impact

Blocker

### Reproducibility

Consistent

https://user-images.githubusercontent.com/6549265/128545904-45b58c93-3563-47af-bc65-e1c6873570c3.mov

**Comment by worldomonation** [comment]
Potentially related: #55149

**Comment by worldomonation** [comment]
Fixed in #55275.
